src/masternodes/rpc_accounts.cpp

```diff
@@ -602,11 +602,7 @@ UniValue utxostoaccount(const JSONRPCRequest& request) {
     fund(rawTx, pwallet, {});
 
     // check execution
-    {
-        LOCK(cs_main);
-        auto metadata = ToByteVector(CDataStream{SER_NETWORK, PROTOCOL_VERSION, msg});
-        execTestTx(CTransaction(rawTx), targetHeight, metadata, CUtxosToAccountMessage{});
-    }
+    execTestTx(CTransaction(rawTx), targetHeight);
```

Comment:
Why this RPC didn't use the second input parameter?

Reply:
There is no auto auth here, when you do `utxostoaccount` it's expected to have coins to do it.